CircleciScripts/run_integrationtest.py

```diff
@@ -6,7 +6,8 @@
 root = sys.argv[2]
 credentials = sys.argv[3]
 testmodules =  ["aws-android-sdk-sqs-test", "aws-android-sdk-transcribe-test", "aws-android-sdk-translate-test",
-                "aws-android-sdk-iot-test", "aws-android-sdk-mobile-client"]
+                "aws-android-sdk-iot-test", "aws-android-sdk-mobile-client", "aws-android-sdk-autoscaling-test",
```

Comment:
can you make one module per line and sort it alphabetically, so it's more readable?